ddp-workspace/projects/ddp-singular/src/app/components/pages/participant-list/participant-list.component.html

```diff
@@ -95,39 +95,36 @@ <h3 class="title-enrollment-message">{{'ParticipantsList.FamilyEnrollmentTitle'
             {{ getParticipantName(participant) }}
           </span>
-            <div class="StatusAndExpandBtn">
+          <div class="StatusAndExpandBtn">
             <div class="enrollmentStatusContainer">
-                <div class="enrollmentStatusIncompleteText" *ngIf="getParticipantStatus(participant) === 'IN_PROGRESS'">
-                    <img class="enrollmentStatusIcon" src="assets/images/alert-circle-orange-512.png">
-                    {{ 'ParticipantStatus.status.inComplete' | translate}}
-                </div>
-                <div class="enrollmentStatusCompleteText" *ngIf="getParticipantStatus(participant) === 'COMPLETE'">
-                    <img class="enrollmentStatusIcon" src="/assets/images/Check.png">
-                    {{ 'ParticipantStatus.status.complete' | translate}}
-                </div>
+              <div class="enrollmentStatusIncompleteText" *ngIf="getParticipantStatus(participant) === 'IN_PROGRESS'">
+                <img class="enrollmentStatusIcon" src="assets/images/alert-circle-orange-512.png">
+                {{ 'ParticipantStatus.status.inComplete' | translate}}
+              </div>
+              <div class="enrollmentStatusCompleteText" *ngIf="getParticipantStatus(participant) === 'COMPLETE'">
+                <img class="enrollmentStatusIcon" src="/assets/images/Check.png">
+                {{ 'ParticipantStatus.status.complete' | translate}}
+              </div>
             </div>
 
+            <div class="participant__icon-container">
+              <button
+                *ngIf="allowParticipantRemoval && !participant.isOperator"
+                mat-icon-button
+                class="participant__delete-btn"
+                (click)="$event.stopPropagation(); onDeleteClick(participant)">
+                <mat-icon>delete</mat-icon>
+              </button>
 
-          <div class="participant__icon-container">
-            <button
-              *ngIf="allowParticipantRemoval && !participant.isOperator"
-              mat-icon-button
-              class="participant__delete-btn"
-              (click)="$event.stopPropagation(); onDeleteClick(participant)"
-            >
-              <mat-icon>delete</mat-icon>
-            </button>
-
-            <button
-              mat-icon-button
-              class="participant__expand-btn"
-              [class.participant__expand-btn--rotate]="isParticipantContentExpanded(participant)"
-            >
-                <div class="hideWord">{{ isParticipantContentExpanded(participant) ? 'Hide' : 'Show' }}</div>
-              <mat-icon>expand_more</mat-icon>
-            </button>
-          </div>
+              <button
+                mat-icon-button
+                class="participant__expand-btn"
+                [class.participant__expand-btn--rotate]="isParticipantContentExpanded(participant)">
+                <span class="hideWord">{{ isParticipantContentExpanded(participant) ? 'Hide' : 'Show' }}</span>
```

Comment:
The only thing I have changed in the file is this line:
replacement `<div class="hideWord">...</div>` with `<span class="hideWord">...</span>`.

Other changes here is just code formatting.